plugins/inputs/monit/monit.go

```diff
@@ -0,0 +1,407 @@
+package monit
+
+import (
+	"encoding/xml"
+	"fmt"
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/internal"
+	"github.com/influxdata/telegraf/internal/tls"
+	"github.com/influxdata/telegraf/plugins/inputs"
+	"golang.org/x/net/html/charset"
+	"net/http"
+)
+
+const (
+	fileSystem string = "0"
+	directory         = "1"
+	file              = "2"
+	process           = "3"
+	remoteHost        = "4"
+	system            = "5"
+	fifo              = "6"
+	program           = "7"
+	network           = "8"
```

Comment:
I know telegraf mostly interests in metric data. Please let me know if you want me to remove the irrelevant service types